rpcs3/Emu/Cell/Modules/cellPad.h

```diff
@@ -192,12 +192,24 @@ struct pad_info
 {
 	atomic_t<u32> max_connect = 0;
 	std::array<u32, CELL_PAD_MAX_PORT_NUM> port_setting{ 0 };
+	std::array<u32, CELL_MAX_PADS> reported_statuses{};
```

Comment:
I don't see a reason why this should be CELL_MAX_PADS.
It's never using more than CELL_PAD_MAX_PORT_NUM.